s3.tf

```diff
@@ -55,3 +59,7 @@ resource "aws_iam_policy" "ses_send_mail_read_s3" {
 output "ses_mailer_bucket" {
   value = "${aws_s3_bucket.ses_mailer_bucket.id}"
 }
+
+output "ses_mailer_bucket_arn" {
```

Comment:
this would be better and more consistent as:
```
output "ses_mailer_bucket" {
  value = {
    id   = "${aws_s3_bucket.ses_mailer_bucket.id}"
    arn = "${aws_s3_bucket.ses_mailer_bucket.arn}"
  }
}
```

Reply:
Happy to do it like that. It will mean re-factoring the code which references this module though, which is why I went for a "minimal-impact" approach originally.

Reply:
This will save a more impacting refactor when we come to need any other attributes, a pain we've been through on other buckets.